massa-client/src/cmds.rs

```diff
@@ -548,13 +548,13 @@ impl Command {
                 }
             }
 
-            Command::get_block => {
+            Command::get_blocks => {
                 if parameters.len() != 1 {
-                    bail!("wrong param numbers")
+                    bail!("wrong param numbers, [IpAddr] shouldn't be empty")
```

Comment:
Error message could be better: Expecting at least one IpAddr ?

Reply:
Errors messages... you know it's HARD, but having some convention would be cool as this one is repetitive.

Reply:
Done